app/controllers/api/groups_controller.rb

```diff
@@ -1,6 +1,12 @@
 class API::GroupsController < API::RestfulController
   load_and_authorize_resource only: :show, find_by: :key
   load_resource only: [:upload_photo, :use_gift_subscription], find_by: :key
+  skip_before_action :authenticate_user!, only: :index
```

Comment:
That's what Rob and I did initially, but it broke all of the tests. If the tests are broken for you, perhaps we could pair on making it work?